u3-views/views/index.hbs

```diff
@@ -0,0 +1,17 @@
+<html>
+
+<body>
+  <h1>U3: Views</h1>
+  <ul>
+    <li><a href="/courses/231370">/courses/:code</a></li>
+    <li><a href="/courses/231370/units">/courses/:code/units</a></li>
+    <li><a href="/courses/231370/students">/courses/:code/students</a></li>
+  </ul>
+  <h1>U3L: Views Labs</h1>
+  <ul>
+    <li><a href="/videos">/videos</a></li>
+    <li><a href="/videos/{{video.id}}">/videos/:id</a></li>
```

Comment:
มุมมองนี้ไม่มีพารามิเตอร์ ไม่มี `video` โปรดใช้ ID ตัวอย่าง เช่น `128`
This view doesn't have parameters. `video` does not exist. Please use an example ID, ex: `128`.